src/v/cluster/topic_table_probe.h

```diff
@@ -51,9 +51,10 @@ class topic_table_probe {
     model::node_id _node_id;
     absl::flat_hash_map<model::topic_namespace, ss::metrics::metric_groups>
```

Comment:
I did not change this as it requires a change to `absl::node_hash_map`, I don't see any concerns with that but wanted to raise the question first.